test/unit/carriers/australia_post_test.rb

```diff
@@ -11,13 +11,13 @@ def setup
   end
 
   def test_service_domestic_simple_request
-    url = 'https://digitalapi.auspost.com.au/postage/parcel/domestic/service.json?from_postcode=2000&height=2&length=19&to_postcode=3108&weight=0.25&width=14'
+    url = 'https://digitalapi.auspost.com.au/postage/parcel/domestic/service.json?from_postcode=2000&height=2.0&length=19.0&to_postcode=3108&weight=0.25&width=14.0'
```

Comment:
Interesting. I don't expect the added `.0` to break things but you never really know.